tools/cldr-apps/src/main/liberty/config/server.xml

```diff
@@ -1,6 +1,7 @@
 <server description="cldr-apps Liberty server">
 	<!-- tag::featureManager[] -->
 	<featureManager>
+		<feature>jsonb-1.0</feature>
```

Comment:
I assume this was needed…? I'd think it would already be available, but anyway this is the right step.

Reply:
I would have thought it wouldn't be needed, but without it I got this:
```
[INFO] (org.codehaus.mojo.pluginsupport.ant.AntHelper) [ERROR   ] SRVE0315E: An exception occurred: java.lang.Throwable: javax.servlet.ServletException: SRVE0207E: Uncaught initialization exception created by servlet
[INFO] (org.codehaus.mojo.pluginsupport.ant.AntHelper) 	at com.ibm.ws.webcontainer.webapp.WebApp.handleRequest(WebApp.java:5107)
[INFO] (org.codehaus.mojo.pluginsupport.ant.AntHelper) 	at [internal classes]
[INFO] (org.codehaus.mojo.pluginsupport.ant.AntHelper) Caused by: javax.servlet.ServletException: SRVE0207E: Uncaught initialization exception created by servlet
[INFO] (org.codehaus.mojo.pluginsupport.ant.AntHelper) 	at com.ibm.ws.webcontainer.servlet.ServletWrapper.init(ServletWrapper.java:368)
[INFO] (org.codehaus.mojo.pluginsupport.ant.AntHelper) 	... 1 more
[INFO] (org.codehaus.mojo.pluginsupport.ant.AntHelper) Caused by: java.lang.NoClassDefFoundError: javax.json.bind.Jsonb
```